sdks/java/core/src/main/java/org/apache/beam/sdk/io/FileIO.java

```diff
@@ -373,9 +373,11 @@ public static Match match() {
   public static MatchAll matchAll() {
     return new AutoValue_FileIO_MatchAll.Builder()
         .setConfiguration(MatchConfiguration.create(EmptyMatchTreatment.ALLOW_IF_WILDCARD))
+            .setAvoidReshuffle(false)
```

Comment:
There are similar configs in different IOs under different names. I recommend use `withOutputParallelization` (used by several IOs e.g. JdbcIO, redisIO, SinglestoreIO)